pkgs/development/libraries/libuv/default.nix

```diff
@@ -81,5 +81,5 @@ in
   }
   //
   mapAttrs (v: h: mkWithAutotools stable (toVersion v) h) {
-    v1_7_5 = "18x6cy2xn31am97vn6jli7kmb2fbp4c8kmv7jm97vggh0x55flsc";
+    v1_9_0 = "0sq8c8n7xixn2xxp35crprvh35ry18i5mcxgwh12lydwv9ks0d4k";
```

Comment:
Any reason to remove the older version? If we're just going to be supporting one version of libuv anyway, we might want to remove this machinery for providing multiple versions.